guides/index_lifecycle.md

```diff
@@ -0,0 +1,152 @@
+# Index Lifecycle
+This guide covers OpenSearch Java Client API actions for Index Lifecycle. You'll learn how to create, read, update, and delete indices in your OpenSearch cluster. We will also leverage index templates to create default settings and mappings for indices of certain patterns.
```

Comment:
```suggestion
This guide covers OpenSearch Java Client API actions for Index Lifecycle. You'll learn how to create, update, and delete indices in your OpenSearch cluster as well as got to index, search and delete the data within the index.
```

Reply:
There is no index template in the guide.

Reply:
According to the description of the [issue](https://github.com/opensearch-project/opensearch-java/issues/428) Java guide similar to ruby [guide](https://github.com/opensearch-project/opensearch-ruby/blob/main/guides/index_lifecycle.md) has to be made, and in this ruby guide there were only crud operations.
If template needs to be added in the guide please help me with some proper documentation meanwhile i followed this [doc](https://opensearch.org/docs/latest/clients/java/) for creating this guide.